projects/packages/changelogger/tests/php/tests/src/UtilsTest.php

```diff
@@ -420,6 +420,95 @@ public function testGetRepoData() {
 		);
 	}
 
+	/**
+	 * Test getRepoData with merge commit.
+	 */
+	public function testGetRepoDataWithMerge() {
```

Comment:
Personally I would now get rid of the `testGetRepoData` test above and just have this new test (renamed to `testGetRepoData`), as it tests all the exact same things plus the new code.

Reply:
makes sense, addressed in https://github.com/Automattic/jetpack/pull/27881/commits/1c43fa3c081c55f017dccbd538655a12d917dc3e